shared/AppInsightsCore/src/JavaScriptSDK/BaseCore.ts

```diff
@@ -231,13 +247,13 @@ export class BaseCore implements IAppInsightsCore {
             };
 
             _self.getPerfMgr = (): IPerfManager => {
-                if (!_perfManager) {
+                if (!_perfManager && !_cfgPerfManager) {
```

Comment:
Will now support the concept of "config" based perf Manager and manually set perf Manager, as once we start allowing unload / re-initialize and config updates we need to track and keep any manually set one.